src/CommunityHydration.js

```diff
@@ -0,0 +1,46 @@
+// const Hydration = require('../src/Hydration');
+
+class CommunityHydration {
+  constructor(data = []) {
+    this.hydrations = data.map(user => new Hydration(user))
+  }
+
+  calculateAvgWaterPerDay(userID) {
+    const userWater = this.hydrations.filter(water => (water.userID === userID));
+    const waterConsumed = userWater.map(water => water.numOunces);
+    const avgWaterConsumed = (waterConsumed.reduce((a, b) => a + b, 0)) / waterConsumed.length;
+    return avgWaterConsumed;
+  }
+
+  calculateTotalWaterOnDay(userID, date) {
+    const userWater = this.hydrations.find(water => water.userID === userID && water.date === date);
+    return userWater.numOunces;
+  }
+
+  convertDateString(date) {
```

Comment:
Great pulling this out to a separate function. I know I use this in my data too, so I was thinking maybe 1) we just map the dates into this format on instantiation, or 2) we have it as a user method or something so we only have to write it once and pass in a user? I don't know the best way to dry this up so maybe we should ask a mentor.

Reply:
@srslie I was wondering about that, too.  I agree-- a good question to ask our mentors.